airrohr-firmware/airrohr-cfg.h

```diff
@@ -46,6 +46,8 @@ enum ConfigShapeId {
 	Config_dnms_read,
```

Comment:
@Warukira looks like you missed this on line one.

// This file is generated, please do not edit.

// Change airrohr-cfg.h.py instead.

Reply:
@chegejames , I did not edit this file. I instead edited airrohr-cfg.h.py, which in turn generated the corresponding changes in this file.